packages/experimental/ActivityIndicator/src/ActivityIndicator.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react';
+import { useRef } from 'react';
+import { View, Animated, Easing, ViewProps } from 'react-native';
+
+/**
+ * let R = big circle radius, from middle of circle to middle of border width = (width - borderRadius) / 2
+ * borderRadius only refers to big circle radius
+ * D = sqrt(2)/2 * R is distance from center to the middle of end circle
+ * Top end circle:
+ *   top: R - D - borderRadius
+ *     top edge is already 1 borderRadius below the big circle
+ *   left: R - D - borderRadius
+ *     left edge is already 1 borderRadius to the right
+ * Bottom end circle:
+ *   top: R + D - 2 * borderRadius
+ *     1 borderRadius explained same as top end circle.
+ *     Extra borderRadius since bottom end circle is 1 borderRadius below top end circle already.
+ *   left: R - D - borderRadius
+ *     same explanation as above
+ */
+// This will be a token later
+// radius is from center to halfway border, border is line thickness
+export const getActivityIndicatorStyle = (radius: number, border: number) => {
+  const xydist = Math.sqrt(2)/2 * radius;
+
+  return {
+    semicircle: {
+      width: radius*2 + border, // from outside edge of circle on one side to the opposite side
+      height: radius*2 + border,
+      borderWidth: border,
+      borderColor: '#919191',
```

Comment:
Since this is used in more than one place, it's preferable to separate it to it's own const variable. Also maybe leave a comment about where you got that exact value (I'm assuming from Figma)

Reply:
I'm not quite sure what you're referring to but I put the `radius*2 + border` in its own variable. The comment describes how I got to that formula based on the other variables.
As for the `borderColor`, yes I got that from Figma and I also separated it out into its own variable.

Reply:
oops. Meant backgroundColor 😅